src/angular/form-elements/checkbox/checkbox.component.html.ts

```diff
@@ -2,7 +2,7 @@ export default `
 <div class="sdc-checkbox">
     <label ripple-click-animation [rippleClickDisabled]="disabled">
         <input type="checkbox" class="sdc-checkbox__input" [ngModel]="checked" (ngModelChange)="toggleState($event)" [disabled]="disabled">
-        <span class="sdc-checkbox__label" *ngIf="label" >{{ label }}</span>
+        <span class="sdc-checkbox__label">{{ label }}</span>
```

Comment:
why did you decide to remove this check? Do you think leaving an empty span is better?

Reply:
Because the checkbox/radio images are now on :before elements of the label, since the actual input is being hidden completely. So the label needs to always be there. There is :empty styling for when there is no content in the label (to remove padding-left).

Reply:
Gotcha